autosklearn/util/logging_.py

```diff
@@ -66,7 +73,9 @@ def __getstate__(self) -> Dict[str, Any]:
         Dictionary, representing the object state to be pickled. Ignores
         the self.logger field and only returns the logger name.
         """
-        return {'name': self.name}
+        return {'name': self.name, 'host': self.host, 'port': self.port,
```

Comment:
Could you please put each key-value pair in a separate line?

Reply:
Done!

Reply:
That one isn't pushed yet, right.